staging/src/k8s.io/apiserver/pkg/storage/testing/watcher_tests.go

```diff
@@ -1492,7 +1492,6 @@ func RunWatchSemantics(ctx context.Context, t *testing.T, store storage.Interfac
 				createdPods = append(createdPods, out)
 			}
 			testCheckResultsInStrictOrder(t, w, scenario.expectedEventsAfterEstablishingWatch(createdPods))
-			testCheckNoMoreResults(t, w)
```

Comment:
This sleeps 1s and with 18 testcases, it just means 18s.

I don't think it adds visible value to have it so I would just remove it for now to get some space here.

Reply:
I would keep it but cut the wait time waaaay down (50ms or 100ms)

Reply:
Good idea - fixed.